src/NServiceBus.Transport.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -48,28 +39,22 @@ class ConnectionFactory
                 throw new ArgumentException("The connectionConfiguration has a null Host.", nameof(connectionConfiguration));
             }
 
-            var requestedHeartbeatToUse = heartbeatInterval != default ? Convert.ToUInt16(heartbeatInterval.TotalSeconds) : connectionConfiguration.RequestedHeartbeat;
-            var networkRecoveryIntervalToUse = networkRecoveryInterval != default ? networkRecoveryInterval : connectionConfiguration.RetryDelay;
-            var sslCertPathToUse = !string.IsNullOrEmpty(clientCertificatePath) ? clientCertificatePath : connectionConfiguration.CertPath;
-            var sslCertPassPhraseToUse = !string.IsNullOrEmpty(clientCertificatePassPhrase) ? clientCertificatePassPhrase : connectionConfiguration.CertPassphrase;
-
-
             connectionFactory = new global::RabbitMQ.Client.ConnectionFactory
             {
                 HostName = connectionConfiguration.Host,
                 Port = connectionConfiguration.Port,
                 VirtualHost = connectionConfiguration.VirtualHost,
                 UserName = connectionConfiguration.UserName,
                 Password = connectionConfiguration.Password,
-                RequestedHeartbeat = requestedHeartbeatToUse,
-                NetworkRecoveryInterval = networkRecoveryIntervalToUse,
+                RequestedHeartbeat = heartbeatInterval ?? connectionConfiguration.RequestedHeartbeat,
+                NetworkRecoveryInterval = networkRecoveryInterval ?? connectionConfiguration.RetryDelay,
                 UseBackgroundThreadsForIO = true
             };
 
             connectionFactory.Ssl.ServerName = connectionConfiguration.Host;
-            connectionFactory.Ssl.Certs = clientCertificates;
-            connectionFactory.Ssl.CertPath = sslCertPathToUse;
-            connectionFactory.Ssl.CertPassphrase = sslCertPassPhraseToUse;
+            connectionFactory.Ssl.Certs = clientCertificateCollection;
+            connectionFactory.Ssl.CertPath = connectionConfiguration.CertPath;
```

Comment:
Oops, I've completely forgotten that those already exists in `connectonConfiguration`.